gimlet/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+helm repo add onechart https://chart.onechart.dev
+
+kubectl create namespace gimlet-installer || true
+
+helm repo update \
+ && helm upgrade gimlet-installer --install onechart/onechart \
+    --namespace gimlet-installer  \
+    -f values.yaml \
```

Comment:
-f values.yaml works from the folder ?

Reply:
I removed the values.yaml and used set for all parameters. The values yaml may not be propagated by the CIVO automation.